setup.py

```diff
@@ -37,11 +37,11 @@ def build_js() -> None:
 with open(requirements_path) as requirements_file:
     requirements = requirements_file.readlines()
 
-__version__ = '3.5.1'
+__version__ = '3.6.0'
 
 oicd = ['flaskoidc==0.1.1']
 pyarrrow = ['pyarrow==3.0.0']
-bigquery_preview = ['google-cloud-bigquery>=2.8.0,<3.0.0', 'flatten-dict==0.3.0']
+bigquery_preview = ['google-cloud-bigquery>=2.13.1,<3.0.0', 'flatten-dict==0.3.0']
```

Comment:
we import this library as `google.cloud`. The `google` package does not exist in v2.8.0. I don't know when exactly it was added but 2.13.1 (latest) has it.